konsave/funcs.py

```diff
@@ -40,7 +40,7 @@ def inner_func(*args, **kwargs):
             function = func(*args, **kwargs)
         except Exception as err:
             dateandtime = datetime.now().strftime("[%d/%m/%Y %H:%M:%S]")
-            log_file = os.path.join(HOME, "konsave_log.txt")
+            log_file = os.path.join(HOME, ".cache/konsave_log.txt")
```

Comment:
Maybe [`$XDG_CACHE_HOME`](https://specifications.freedesktop.org/basedir-spec/latest/ar01s03.html) (plus parse colons) with a fallback? Eh, maybe not worth it. (Still an improvement.)

Reply:
I've just spot checked my Arch and Ubuntu 22.04 machines and $XDG_CACHE_HOME so opted to hard code $HOME/.cache. Happy to put some more fallback logic in but I'm not sure its worth it.

Reply:
Thanks! Moving the log to the `.cache` folder seems nice! I'll merge it :)